web/ts/splitview.ts

```diff
@@ -52,15 +56,38 @@ export class SplitView {
         return [100 - this.camPercentage, this.camPercentage];
     }
 
-    private toggleControlBars(camPercentage: number) {
-        let i = 0,
-            j = 1;
-        if (camPercentage > 50) {
-            (i = 1), (j = 0);
+    private cloneEvents(srcElem: HTMLElement, destElem: HTMLElement, events: string[]) {
```

Comment:
I think this function could also be useful elsewhere, so we might want to move it to the `global.ts` file.

Reply:
Good catch! Also kind of bad is that you can only fullscreen one of the two players. You would probably expect to fullscreen the whole split view ... may also something for a followup!

Reply:
@MatthiasReumann https://github.com/joschahenningsen/TUM-Live/pull/895 lets the user make the splitview fullscreen